pkgs/applications/virtualization/youki/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, installShellFiles
+, dbus
+, libseccomp
+, systemd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "youki";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-XwapCfu6Me0xSe+qFz9BFRYpQvG+ztb6QyhGejYRPb4=";
+  };
+
+  buildInputs = [ dbus libseccomp systemd ];
+  nativeBuildInputs = [ pkg-config installShellFiles ];
```

doesn't really matter but usually `nativeBuildInputs` comes before `buildInputs`
```suggestion
  nativeBuildInputs = [ pkg-config installShellFiles ];

  buildInputs = [ dbus libseccomp systemd ];
```

---

I fixed it, thanks